hoomd/hpmc/update.py

```diff
@@ -521,71 +407,56 @@ class Clusters(Updater):
-            may be swapped.
         move_ratio (float): Set the ratio between pivot and reflection moves.
         flip_probability (float): Set the probability for transforming an
                                  individual cluster.
-        swap_move_ratio (float): Set the ratio between type swap moves and
-                                geometric moves.
         trigger (Trigger): Select the timesteps on which to perform cluster
             moves.
 
     The GCA as described in Liu and Lujten (2004),
     http://doi.org/10.1103/PhysRevLett.92.035504 is used for hard shape, patch
     interactions and depletants.
 
-    With depletants, Clusters are defined by a simple distance cut-off
-    criterion. Two particles belong to the same cluster if the circumspheres of
-    the depletant-excluded volumes overlap.
-
-    Supported moves include pivot moves (point reflection), line reflections
-    (pi rotation around an axis), and type swaps.  Only the pivot move is
-    rejection free. With anisotropic particles, the pivot move cannot be used
-    because it would create a chiral mirror image of the particle, and only
-    line reflections are employed. Line reflections are not rejection free
-    because of periodic boundary conditions, as discussed in Sinkovits et al.
-    (2012), http://doi.org/10.1063/1.3694271.
+    Implicit depletants are supported and simulated on-the-fly, as if they were
+    present in the actual system.
 
+    Supported moves include pivot moves (point reflection) and line reflections
+    (pi rotation around an axis).  With anisotropic particles, the pivot move
+    cannot be used because it would create a chiral mirror image of the
+    particle, and only line reflections are employed. In general, line
+    reflections are not rejection free because of periodic boundary conditions,
+    as discussed in Sinkovits et al. (2012), http://doi.org/10.1063/1.3694271 .
+    However, we restrict the line reflections to axes parallel to the box axis,
+    which makes those moves rejection-free for anisotropic particles, but the
+    algorithm is then no longer ergodic for those and needs to be combined with
+    local moves.
 
-    Note:
-        The type swap move only works between two types of equal sized,
-        unorientable (``orientable``=``False``) spherical particles and
-        exchanges their identities.
 
     .. rubric:: Threading
 
     The `Clusters` updater support threaded execution on multiple CPU cores.
 
     Attributes:
         seed (int): Random number seed.
-        swap_type_pair (list): A pair of two types whose identities may be swapped.
-        delta_mu (float): The chemical potential difference between types to be swapped
-        move_ratio (float): Set the ratio between pivot and reflection moves.
-        flip_probability (float): Set the probability for transforming an
+        move_ratio(float): Set the ratio between pivot and reflection moves.
```

Comment:
@vyasr pointed out that `move_ratio` parameters should be renamed #777.

Reply:
done for clusters updater